### PR TITLE
add memo,evidence_status field to PO

### PR DESCRIPTION
**Pull request by SokreamPhanqrl**
@yostashiro For PDF attachment on PO list view, I get the number of attachments from messages. Is that exactly what the customer want?

### CODE REVIEW

**Comment by yostashiro** [comment]
@SokreamPhanqrl Not exactly.  Please follow the design here: https://github.com/odoo/odoo/blob/cf54606edb9130f2c5b351150f0e37b75f10ae66/addons/hr_expense/views/hr_expense_views.xml#L511-L512

File: purchase_order_adj_asx/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class PurchaseOrder(models.Model):
+   _inherit = 'purchase.order'
+
+   evidence_status = fields.Selection([
+      ('more', 'Need More'),
+      ('done', 'Done'),
+   ], string="Evidence Status", default='')
+    
+   evidence_text = fields.Char()
+   memo = fields.Text(string="Memo")
```
Comment (yostashiro) [inline purchase_order_adj_asx/models/purchase_order.py]: No need to add string here if the field is one word.

File: purchase_order_adj_asx/views/purchase_order_views.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <field name="name">purchase.order.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='origin']" position="after">
+                <field name="memo"/>
+                <field name="evidence_status"/>
+                <field name="evidence_text"/>
+                <field name="message_attachment_count" string="Attachments"/>
+                <button name="action_get_attachment_view" string="View Attachments" type="object" icon="fa-paperclip"/>
```
Comment (yostashiro) [inline purchase_order_adj_asx/views/purchase_order_views.xml]: @SokreamPhanqrl Probably better to remove the strings from these fields, following the style of expense sheet.  What do you think?

**Review by yostashiro — COMMENTED** [review]
Please follow up on some minor adjustments, install the module in the test database and let me know.  Thanks!

File: purchase_order_adj_asx/views/purchase_order_views.xml (hunk 2 above)
Comment (SokreamPhanqrl) [reply purchase_order_adj_asx/views/purchase_order_views.xml]: It's okay if user knows what those number are about , but what if they don't(for the first time). But anyway, it is better to remove it. :D

File: purchase_order_adj_asx/models/purchase_order.py (hunk 1 above)
Comment (SokreamPhanqrl) [reply purchase_order_adj_asx/models/purchase_order.py]: ok

**Pull request merged**